timescaledb/how-to-guides/analytics/approximate-percentile.md

```diff
@@ -0,0 +1,68 @@
+# Approximate percentiles
+In general, percentiles are useful for understanding the distribution of data.
+The 50th percentile is the point at which half of your data is greater and half
+is lesser. The 10th percentile is the point at which 90% of the data is greater,
+and 10% is lesser. The 99th percentile is the point at which 1% is greater, and
+99% is lesser.
+
+The 50th percentile, or median, is often a more useful measure than the average,
+especially when your data contains outliers. Outliers can dramatically change
+the average, but do not effect the median as much. For example, if you have
+three rooms in your house and two of them are 40℉ (4℃) and one is 130℉ (54℃),
+the fact that the average room is 70℉ (21℃) doesn't matter much. However, the
+50th percentile temperature is 40℉ (4℃), and tells you that at least half your
+rooms are at refrigerator temperatures (also, you should probably get your
+heating checked!)
+
+Percentiles are sometimes used less frequently because they can use more CPU and
+memory to calculate than an average or another aggregate measure. This is
+because an exact computation of the percentile needs the full dataset as an
+ordered list. Timescale Analytics uses approximation algorithms to calculate a
+percentile without requiring all of the data. This also makes them more
+compatible with continuous aggregates. By default, Timescale Analytics uses
```

Comment:
I'll probably want to take a pass at this, I like a lot of the edits you've been doing here, but I think some may elide differences that are important, will have to look a little deeper at some point...

Reply:
Yep, it definitely could use some help from a subject matter expert!